### DISCUSSION TITLE
How to get the total number item in a ListPart by code?

### DISCUSSION THREAD

**Discussion by hylinux** [OrchardCMS/OrchardCore#13670] [Q&A] [answered]
Hi,  team,  still I can not find out a document which can help me to get the total number item in a ListPart by code.

if you know the way in code? would you share it with me?

thanks

btw: context:
I create a `ArticlePage` ContentType with a ListPart.  I wrote code in template file: `Content-ArticlePage.cshtml`:

```csharp

ContentItem contentItem = Model.ContentItem;
ListPart list = contentItem.As<ListPart>();
/* I got a ListPart instance right now, but I don't know which method can help me get the total number in this list */

```

thanks

**Comment by hishamco** [comment, marked as answer]
You could use `IOrchardHelper.QueryListItemsCountAsync()`

**Reply by hylinux** [reply]
copy. thanks for your sharing.  let me try it.  thanks.

**Reply by hishamco** [reply]
Is it solve your issue?

**Reply by hylinux** [reply]
yep. thanks again.

**Comment by hishamco** [comment]
You could use `IOrchardHelper.QueryListItemsCountAsync()`
